tests/safeds/data/tabular/containers/_table/test_get_similar_columns.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.exceptions._data import UnknownColumnNameError
+
+
+@pytest.mark.parametrize(
+    ("table", "column_name", "expected"),
+    [
+        (Table({"column1": ["col1_1"], "x": ["y"], "cilumn2": ["cil2_1"]}), "col1", ["column1"]),
+        (
+            Table(
+                {
+                    "column1": ["col1_1"],
+                    "col2": ["col2_1"],
+                    "col3": ["col2_1"],
+                    "col4": ["col2_1"],
+                    "cilumn2": ["cil2_1"],
+                },
+            ),
+            "clumn1",
+            ["column1", "cilumn2"],
+        ),
+    ],
+    ids=["one similar", "two similar/ dynamic increase"],
+)
```

Comment:
We also need a case for an empty table.

And I guess it also makes sense to have a test case that *returns* an empty list.

Reply:
Thanks. It will be updated.